azure-pipelines/util/appimage_apprun.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+#
+# Custom AppRun for Xournal++
+
+export PATH="$APPDIR/usr/bin/:$PATH"
```

Comment:
Escaping here is suboptimal. Use `export PATH="$APPDIR"/usr/bin:"$PATH"`.

For the other uses, prefer quoting variables only as far as possible, non-special characters don't need quoting.

You should make sure `$APPDIR` is available when you e.g., want to use your script from an extracted AppImage. It won't run right now as-is.
Example code: `export APPDIR=${APPDIR:-$(readlink -f $(dirname "$0"))}`